keras_hub/src/utils/timm/preset_loader.py

```diff
@@ -16,9 +16,24 @@ class TimmPresetLoader(PresetLoader):
     def __init__(self, preset, config):
         super().__init__(preset, config)
         architecture = self.config["architecture"]
-        if "resnet" in architecture:
+        if architecture in (
+            "resnet18",
+            "resnet26",
+            "resnet34",
+            "resnet50",
+            "resnetv2_50",
+            "resnet101",
+            "resnetv2_101",
+            "resnet152",
+            "resnetv2_152",
+        ):
             self.converter = convert_resnet
-        elif "csp" in architecture:
+        elif architecture in (
```

Comment:
can we just flip the order of the if and else if? This will be hard to maintain.

Reply:
Makes sense, this would be lot better, made the changes. Thank you.